app/controllers/idv/capture_doc_status_controller.rb

```diff
@@ -12,7 +12,7 @@ def show
 
     def status
       @status ||= begin
-        if !flow_session || !document_capture_session
```

Comment:
@jmhooper I think you worked in this file recently--now that the barcode stuff and document_capture_session_uuid are out of flow_session, I think we can remove all references to flow_session from this file (see 18562aa7eb9bfff1e16452d1306d2a6a39a10721)--any issues with this?